core-bundle/contao/templates/twig/component/_style.html.twig

```diff
@@ -0,0 +1,23 @@
+{#
```

Comment:
Did you accidentally commit this file or where is this used? :slightly_smiling_face:

Reply:
It's not used by Contao - but its addition was no accident.

Reply:
Its content was an accident though 😁 fixed now.